### PR TITLE
feature(argus): Update to 0.11.7

### PR DESCRIPTION
**Pull request by k0machi**
Just an update to sync argus version with master

### Testing
- [x] Job [here](https://argus.scylladb.com/test/3ff64cbc-43d6-4163-a8d9-fa56529a4dc9/runs?additionalRuns[]=5adb2958-9b74-4be7-8f05-d5ac3aab7488)
 
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: docker/env/requirements.txt
Comment (fruch) [inline docker/env/requirements.txt:1]: why adding this file ? you want to update the one on the root directory

Comment (k0machi) [reply docker/env/requirements.txt:1]: > why adding this file ? you want to update the one on the root directory

not sure where it came from, removing

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-1.38-argus-backport-perf-v14
+1.39-argus-update-0.11.7
```
Comment (vponomaryov) [inline docker/env/version:1]: Current `Hydra` image for the `master` branch of the SCT is `1.54-update-k8s-components-1`.
This one is made for the `perf-v14` specifically.
So, please update it the way we could easily distinguish the master and perf branch hydra images.

**Comment by k0machi** [comment]
Link to [Argus](https://argus.scylladb.com/test/3ff64cbc-43d6-4163-a8d9-fa56529a4dc9/runs?additionalRuns[]=5adb2958-9b74-4be7-8f05-d5ac3aab7488)

File: docker/env/version (hunk 1 above)
Comment (k0machi) [reply docker/env/version]: Done

**Review by soyacz — APPROVED** [review]
LGTM

**Review by vponomaryov — APPROVED** [review]
LGTM

**Pull request merged**